Makefile

```diff
@@ -106,7 +106,7 @@ bootstrap-database:
 
 .PHONY: reindex
 reindex:
-	$(CONSOLE) akeneo:elasticsearch:reset-indexes
```

Comment:
Great! I think this was in here cause this was part of the *install akeneo* process (which does need to re-create them).

I think we should probably have another Make target that resets the indexes first and then does this `rebuild_indexes` or some such. That should be triggered wherever this is currently triggered in the makefile.

Reply:
Thanks for the feedback! Maybe we could leave the current Make target the same and add a new one that we can reference in the crontab. Something like:

```make
.PHONY: reindex
reindex:
	$(CONSOLE) akeneo:elasticsearch:reset-indexes
	$(MAKE) reindex-in-place

.PHONY: reindex-in-place
reindex-in-place:
	$(CONSOLE) pim:product:clean-removed-products
	$(CONSOLE) pim:product-model:index --all
	$(CONSOLE) pim:product:index --all
```

Reply:
Perfect!